### PR TITLE
docs: Describe using Alpine-based Fluentd image

### LINKED ISSUE #1703: 1.12.2-sumo image critical and high vulnerabilities
The latest fluentd image version for the sumologic-kubernetes-collection chart (chart version 2.1.5, fluentd image version public.ecr.aws/sumologic/kubernetes-fluentd:1.12.2-sumo-2), has 1 critical vulnerability and 7 high vulnerabilities, found by aws ecr vulnerability scans. We are currently working to drive both of these categories to 0 in images running in our kubernetes cluster. Is there a time line where we can expect these to be addressed in your image? And does sumo have a plan to keep these categories at 0 (or as close as possible as new issues are found) going forward?

### PR DESCRIPTION
**Pull request by astencel-sumo**
Fixes #1703

### CODE REVIEW

File: deploy/docs/Best_Practices.md [hunk 1]
```diff
@@ -103,6 +104,42 @@ to the `Docker_Mode_Parser` parameter in the `Input plugin` configuration of flu
...
+
+For example:
+
+```yaml
+fluentd:
+  image:
+    tag: 1.12.2-sumo-4-alpine
+```
+
+Go to the [official Fluentd image repository](https://gallery.ecr.aws/sumologic/kubernetes-fluentd)
```
Comment (sumo-drosiek) [inline deploy/docs/Best_Practices.md:131]:
```suggestion
Go to the [official Sumo Logic's Fluentd image repository](https://gallery.ecr.aws/sumologic/kubernetes-fluentd)
```

File: deploy/docs/Best_Practices.md [hunk 2]
```diff
@@ -103,6 +104,42 @@ to the `Docker_Mode_Parser` parameter in the `Input plugin` configuration of flu
...
+to find the latest release of Fluentd.
+The Alpine-based releases are the ones with the `-alpine` suffix.
+
+Both Debian-based and Alpine-based images support the same architectures:
+
+- x86-64,
+- ARM 32-bit,
+- ARM 64-bit.
+
+The source code and the `Dockerfile`s for both images can be found at https://github.com/SumoLogic/sumologic-kubernetes-fluentd.
```
Comment (sumo-drosiek) [inline deploy/docs/Best_Practices.md:141]:
```suggestion
The source code and the `Dockerfile`s for both images can be found at [https://github.com/SumoLogic/sumologic-kubernetes-fluentd](https://github.com/SumoLogic/sumologic-kubernetes-fluentd).
```

Comment (astencel-sumo) [reply deploy/docs/Best_Practices.md:141]: Oh I don't think that's needed when the URL and the text are the same. GitHub (and I think any Markdown viewer) shows this as a link anyway.

Comment (sumo-drosiek) [reply deploy/docs/Best_Practices.md:141]: As long as mdl is happy I'm fine :)

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**